dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -184,12 +212,32 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
                 break;
             } else {
                 log.info("Waiting for GitHub API cache to clear...");
-                Thread.sleep(TimeUnit.MINUTES.toMillis(1));
```

Comment:
Why not just do `TimeUnit.MNUTES.sleep()`?

Reply:
The extraction of `waitFor` is mostly for improvements in unit tests (you don't actually have to wait in the tests). I had considered `waitFor(seconds)` but thought that kind of bucks normal standards of millis and allows the caller to be somewhat explicit with `TimeUnit.${UNIT}.toMillis(${NUM})`. I also considered `waitForMinutes()`, `waitForSeconds()`, etc... but then thought I was going too far and this seemed to express things.